### PR TITLE
Add more tests

### PR DESCRIPTION
**Pull request by ivanceras**
Some of these test are obvious, but helps cement the behavior of the functions.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1149,6 +1149,131 @@ mod tests {
...
+                multi_id,
+                None,
+                hash.clone()
+            ));
+            assert_ok!(MultiAccount::approve(
+                Origin::signed(2),
+                multi_id,
+                Some(now()),
+                hash.clone()
+            ));
```
Comment (philipstanislaus) [inline src/lib.rs]: This second approve is not necessary for the test

**Review by philipstanislaus — APPROVED** [review]
Thanks!

**Comment by lucasvo** [comment]
@ivanceras could you fix the build and get this ready to merge?

File: src/lib.rs (hunk 1 above)
Comment (lucasvo) [reply src/lib.rs]: @ivanceras could you fix this?

**Comment by ivanceras** [comment]
@lucasvo I don't have a merge previlege on this repo. 
@philipstanislaus can you grant previlege to merge PR?

File: .travis.yml [hunk 2]
```diff
@@ -28,7 +28,6 @@ before_script:
 script:
 - cargo build
 - cargo test
-- cargo fmt -- --check
```
Comment (philipstanislaus) [inline .travis.yml]: Why did we remove this check?

**Review by philipstanislaus — CHANGES_REQUESTED** [review]

**Comment by philipstanislaus** [comment]
@ivanceras thanks for the changes. Why did we remove the `cargo fmt` check in CI?

**Comment by lucasvo** [comment]
@ivanceras fixed access rights.

File: .travis.yml (hunk 2 above)
Comment (ivanceras) [reply .travis.yml]: my bad, it was meant for the other [pallet](https://travis-ci.com/github/centrifuge/substrate-pallet-nft-registry) which ci was causing errors(which I thought was `cargo fmt -- --check`).

**Review by philipstanislaus — APPROVED** [review]
Thanks for adding the linter back – feel free to merge once tests pass!

**Comment by ivanceras** [comment]
@philipstanislaus the `cargo fmt-- --check` is indeed causing the ci script to [fail](https://travis-ci.com/github/centrifuge/substrate-pallet-multi-account/builds/165207787).


```bash
Diff in /home/travis/build/centrifuge/substrate-pallet-multi-account/src/lib.rs at line 1246:
                 Some(now()),
                 hash.clone()
             ));
-            assert_ok!(
-                MultiAccount::cancel(Origin::signed(1), multi_id, now(), hash.clone()),
-            );
+            assert_ok!(MultiAccount::cancel(
+                Origin::signed(1),
+                multi_id,
+                now(),
+                hash.clone()
+            ),);
         });
     }
 
The command "cargo fmt -- --check" exited with 1.
```
I guess we will be using `cargo fmt` for this repo.

**Comment by ivanceras** [comment]
@lucasvo I can't still merge PR.

**Pull request closed without merging**